.eslintrc

```diff
@@ -196,7 +198,6 @@
     "space-before-function-paren": 0,
     "space-in-parens": [1, "never"],
     "space-infix-ops": 1,
-    "space-return-throw-case": 1,
```

Comment:
Ah, that's right:

`Replacement notice: This rule was removed in ESLint v2.0 and replaced by keyword-spacing rule.`

These rules were removed.